src/jemalloc.c

```diff
@@ -81,6 +81,17 @@ const char *zero_realloc_mode_names[] = {
 	"abort",
 };
 
+static void default_junk_alloc(void *ptr, size_t usize) {
+	memset(ptr, 0xa5, usize);
+}
+
+static void default_junk_free(void *ptr, size_t usize) {
+	memset(ptr, 0x5a, usize);
+}
+
```

Comment:
Explicitly define the two junk constants?

Reply:
Done.